### PR TITLE
Enablereturn button

### PR DESCRIPTION
**Pull request by tildedash**

### CODE REVIEW

File: controllers/admin/AdminShoprunbackController.php [hunk 1]
```diff
@@ -169,10 +174,20 @@ public function initContent()
             'srbtoken'                          => RestClient::getClient()->getToken(),
             'link'                              => $link,
             'template'                          => $template,
-            'admin_module_ajax_url_shoprunback' => 'index.php?controller=AdminShoprunback&token=' . Tools::getAdminTokenLite('AdminShoprunback')
+            'admin_module_ajax_url_shoprunback' => 'index.php?controller=AdminShoprunback&token=' . Tools::getAdminTokenLite('AdminShoprunback'),
+            'enable_return_btn' => (Configuration::get('SRB_ENABLE_RETURN_BTN') !== false) ? Configuration::get('SRB_ENABLE_RETURN_BTN') : '0'
```
Comment (tildedash) [inline controllers/admin/AdminShoprunbackController.php]: does Configuration::get('SRB_ENABLE_RETURN_BTN') always got a value (boolean).

If yes, reverse the condition and remove the !== 

`
Configuration::get('SRB_ENABLE_RETURN_BTN')  ?  '0' : Configuration::get('SRB_ENABLE_RETURN_BTN')
`

File: views/templates/front/orderDetail.tpl [hunk 2]
```diff
@@ -23,6 +23,11 @@
  *}
  
  <section class="box" id="order-detail">
+
+    {if isset($token_in_url) && $token_in_url === true}
```
Comment (tildedash) [inline views/templates/front/orderDetail.tpl]: === true is facultative

**Comment by tildedash** [comment]
@MrBaiame I have created the PR for you ;)

File: controllers/admin/AdminShoprunbackController.php (hunk 1 above)
Comment (MrBaiame) [reply controllers/admin/AdminShoprunbackController.php]: that configuration would mean that if it has a value, we take 0, isn't it ? 
Configuration::get('SRB_ENABLE_RETURN_BTN') ? Configuration::get('SRB_ENABLE_RETURN_BTN') : '0' would be better ? I am doubting all of a sudden !

File: views/templates/front/orderDetail.tpl (hunk 2 above)
Comment (MrBaiame) [reply views/templates/front/orderDetail.tpl]: token_in_url has the "false" value by default in the hookDisplayOrderDetail, i think if we remove the === true condition, we would go into the "if" when we don't want to ?

File: controllers/admin/AdminShoprunbackController.php (hunk 1 above)
Comment (tildedash) [reply controllers/admin/AdminShoprunbackController.php]: you make me doubting now :p 

If after testing, it works, I'm ok with that

File: views/templates/front/orderDetail.tpl (hunk 2 above)
Comment (tildedash) [reply views/templates/front/orderDetail.tpl]: ok

**Pull request closed without merging**